build/nuke/Build.Steps.cs

```diff
@@ -251,6 +256,36 @@ partial class Build
             );
         });
 
+    Target CopyAdditionalDeps => _ => _
+        .Unlisted()
+        .Description("Creates AdditionalDeps and shared store in tracer-home")
+        .After(CompileManagedSrc)
+        .Executes(() =>
+        {
+            DotNetPack(s => s
+                .SetProject(Solution.GetProject(Projects.AdditionalDeps))
+                .SetConfiguration(BuildConfiguration)
+                .EnableNoBuild()
+                .EnableNoRestore()
+                .SetOutputDirectory(AdditionalDepsDirectory));
+
+            DotNetPublish(s => s
+                .SetProject(Solution.GetProject(Projects.AdditionalDeps))
+                .SetConfiguration(BuildConfiguration)
+                .SetTargetPlatformAnyCPU()
+                .SetProperty("TracerHomePath", (TracerHomeDirectory / TargetFramework.NETCOREAPP3_1).ToString())
+                .EnableNoBuild()
+                .EnableNoRestore()
+                .CombineWith(TestFrameworks.ExceptNetFramework(), (p, framework) => p
+                    .SetFramework(framework)
+                    .SetOutput(AdditionalDepsDirectory / "shared" / "Microsoft.NETCore.App" / framework.ToString()[^3..] + ".0")));
```

Comment:
Maybe just me but `framework.ToString()[^3..] + ".0"` seems a bit cryptic. Also the additional .0 that seems unusual in first sight. A comment might help there to expalin.